hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -130,6 +131,7 @@ public abstract class BaseHoodieWriteClient<T, I, K, O> extends BaseHoodieClient
   protected static final String LOOKUP_STR = "lookup";
   private static final long serialVersionUID = 1L;
   private static final Logger LOG = LoggerFactory.getLogger(BaseHoodieWriteClient.class);
+  private static final String SPARK_COMPLEX_KEYGEN_CLASS_NAME = "org.apache.hudi.keygen.ComplexKeyGenerator";
```

Comment:
This base hoodie write client is used by other engines outside of spark, so wondering if this naming should be something engine agnostic?